src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -167,6 +205,8 @@ def execute(self, reqStatus):
 
         # Call _execute() and feed the relevant pipeline with the objects popped from requestRecords
         try:
+            self.getGlobalLocks()
+            # self.logger.debug("GlobalLocks: %s", pformat(self.globalLocks))
```

Comment:
TODO: reminder to remove this debugging line before merging this pr.

Reply:
Done